ui/components/CategoryCard.js

```diff
@@ -0,0 +1,70 @@
+import { Card, CardActions, CardHeader, IconButton } from "@mui/material";
+import AddIcon from '@mui/icons-material/Add';
+import DeleteIcon from '@mui/icons-material/Delete';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+
+function CategoryCard({cat}) {
+  if (typeof cat === "undefined") {
+    cat = 0;
+  }
+  let content;
+  let description;
+  switch (cat) {
+    case 0:
+      content = "Manage Service Mesh Lifecycle";
+      description = "Deploy a service mesh or SMI adapter on your cluster.";
+      break;
+
+    case 1:
+      content = "Manage Sample Application Lifecycle";
+      description = "Deploy sample applications on/off the service mesh.";
+      break;
+
+    case 2:
+      content = "Apply Service Mesh Configuration";
+      description = "Configure your service mesh using some pre-defined options.";
+      break;
+
+    case 3:
+      content = "Validate Service Mesh Configuration";
+      description = "Validate your service mesh configuration against best practices.";
+      break;
+
+    case 4:
+      content = "Apply Custom Configuration";
+      description = "Customize the configuration of your service mesh.";
+      break;
+  }
+  return (
+    <Card style={
```

Comment:
Use `sx` here instead of `style` as this is a mui component

Reply:
Are we going to ```sx```  just for ```Card``` or to other components as well

Reply:
Yup use `sx` for all the mui components, and try to use mui components as much as possible , like you can use `box` instead of `div`